iosApp/iosApp/Pages/NearbyTransit/NearbyViewModel.swift

```diff
@@ -63,7 +63,7 @@ class NearbyViewModel: ObservableObject {
     }
 
     func goBack() {
-        navigationStack.removeLast()
+        _ = navigationStack.popLast()
```

Comment:
Where will the user be directed when the navigation stack is empty? Is it worth having a default route we use when the user triggers this behavior?

Reply:
There kinda is a default route already, in `ContentView` line 163 we use `lastSafe()` which will return nearby route if there is no route.